Formula/kustomize.rb

```diff
@@ -23,7 +23,7 @@ class Kustomize < Formula
 
   def install
     commit = Utils.git_head
-    tag = Utils.safe_popen_read("git", "tag", "--contains", "HEAD").strip
+    tag = Utils.safe_popen_read("git", "tag", "--points-at", "HEAD", "--sort", "-version:creatordate").strip
```

Comment:
We can. I wasn't sure if there was an exact reason for doing this in the first place.
I guess it could have been to match how `goreleaser` would generate version info.

It will also require modifing the test as testcase checks for the full tag name (e.g. `kustomize/v4.1.2`)

Reply:
Or we could just pass `kustomize/v#{version}` to `sigs.k8s.io/kustomize/api/provenance.version` so we don't need to modify the test.